bindings/wasm/bindings.d.ts

```diff
@@ -0,0 +1,39 @@
+type Matrix1x3 = [number, number, number];
+type Matrix4x3 = [Matrix1x3, Matrix1x3, Matrix1x3, Matrix1x3];
+type Vec3 = [number, number, number];
+type Vec3Fun = (v: Vec3) => Manifold;
+type Vec2 = [number, number];
+type SimplePolygon = Vec2[];
+type Polygons = SimplePolygon[];
+
+declare class Manifold {
+  transform(matrix: Matrix4x3): Manifold;
+
+  translate: Vec3Fun;
+  rotate: Vec3Fun;
+  scale: Vec3Fun;
+  add(other: Manifold): Manifold;
+  subtract(other: Manifold): Manifold;
+  intersect(other: Manifold): Manifold;
+  refine(n: number): Manifold;
+}
+
+declare function cube(size: Vec3, center?: boolean): Manifold;
+declare function cube(size?: number, center?: boolean): Manifold;
```

Comment:
Is this something we can do in the C++ API? I sort of thought it would be just `cube(size?: Vec3, center?: boolean): Manifold`

Reply:
Ah, right. Yeah, let's stick with self-consistency for now. Hopefully it won't make too much difference when we eventually try to translate from OpenSCAD files.